File: src/main/java/com/challenge/interview/calendar/api/dto/AvailabilityDto.java
package com.challenge.interview.calendar.api.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.OffsetDateTime;

@Data
@AllArgsConstructor
public class AvailabilityDto {
    private Long id;
    private OffsetDateTime from;
    private OffsetDateTime to;
}

File: src/main/kotlin/com/challenge/interview/calendar/api/dto/AvailabilityDto.kt
package com.challenge.interview.calendar.api.dto

import java.time.OffsetDateTime

class NonIdentifiableAvailabilityDto(val from: OffsetDateTime, val to: OffsetDateTime);
class AvailabilityDto(val id: Long, val from: OffsetDateTime, val to: OffsetDateTime);